src/LineChart.ts

```diff
@@ -34,6 +34,7 @@ module n3Charts {
         ['container', Factory.Container, element[0]],
         ['x-axis', Factory.Axis, 'x'],
         ['y-axis', Factory.Axis, 'y'],
+        ['lines-container', Factory.Series.LineSeriesContainer, 'y'],
```

Comment:
Maybe we could let `Factory.Container` make a `data` g node, and then every `Factory.Series.Line`, `Factory.Series.Area` and so on creates a g node for itself. Like this we could reduce the number of factories.